idioms/ootb-dyn-dispatch.md

```diff
@@ -0,0 +1,83 @@
+# On-Stack Dynamic Dispatch
```

Comment:
I think this is more well-known as out-of-box dynamic dispatch from blog posts, maybe we could mention it as well?

Reply:
I don't think that will be necessary.